cli/internal/analytics/client.go

```diff
@@ -20,9 +20,10 @@ var (
 )
 
 type eventDetails struct {
-	user        cqapi.User
-	currentTeam string
-	environment string
+	user                cqapi.User
+	currentTeam         string
+	currentTeamInternal bool
```

Comment:
I would call this `isCurrentTeamInternal`. It's just clearer what it is when the type is not immediately visible.